conda-dist/recipes/threeml/meta.yaml

```diff
@@ -55,6 +55,8 @@ requirements:
     - interpolation>=2.1.5 # [py3k]
     - tqdm
```

Comment:
tqdm=4.56.0 was needed for me to work with the colored progress bars

Reply:
Thanks!